src/app/errors/components/OuterErrorBoundary.tsx

```diff
@@ -18,11 +18,13 @@ const OuterErrorBoundary = (props: { children?: React.ReactNode }) => {
       setIntl(intlObject);
     };
 
-    setUpIntl();
-  }, []);
+    if (!intl) {
+      setUpIntl();
```

Comment:
OK with no await here?

Reply:
Yea it shouldn't matter, this is the way it was set up already